go.mod

```diff
@@ -49,6 +49,7 @@ require (
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6 // indirect
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	gopkg.in/yaml.v2 v2.2.7 // indirect
+	honnef.co/go/tools v0.0.0-20190523083050-ea95bdfd59fc // indirect
```

Comment:
I am not sure if this change is needed.

Reply:
I have no idea. In all my recent PRs, `git commit -a` or something else automatically adds this change.

Reply:
@tonyyang-svail I verified on develop branch.  `go generate ./...` introduces this change.

Reply:
I see. Looks like `linter` depends on it. https://pkg.go.dev/mod/honnef.co/go/tools@v0.0.1-2020.1.4